lightning/src/ln/functional_tests.rs

```diff
@@ -7826,6 +7853,11 @@ fn test_bump_penalty_txn_on_revoked_htlcs() {
 
 		check_spends!(node_txn[0], revoked_local_txn[0]);
 		check_spends!(node_txn[1], revoked_local_txn[0]);
+		// Note that these are both bogus - they spend outputs already claimed in block 129:
+		assert!((node_txn[0].input[0].previous_output == revoked_htlc_txn[0].input[0].previous_output &&
+		         node_txn[1].input[0].previous_output == revoked_htlc_txn[1].input[0].previous_output) ||
+		        (node_txn[0].input[0].previous_output == revoked_htlc_txn[1].input[0].previous_output &&
+		         node_txn[1].input[0].previous_output == revoked_htlc_txn[0].input[0].previous_output));
```

Comment:
Is there any way to control the `||` by an if-else condition where each has two separate asserts representing the `&&`? It would make a failing test easier to understand.